### ISSUE TITLE
1.19.1 support?

### ISSUE THREAD

**Issue by 19lmyers** [stashymane/extra-sounds#71]
I noticed the mod does not work on 1.19.1. How feasible is it for support to be added?

**Comment by lonefelidae16** [comment]
Same here, I can’t wait to 1.19.1 support.
So I have wrote the patch and I’ll share my code.

**Comment by ttrafford7** [comment]
does your instance crash while trying to launch it?

**Comment by akemin-dayo** [comment]
I've added support for Minecraft 1.19.2 and 1.19.1 [in my fork of ExtraSounds](https://github.com/akemin-dayo/extra-sounds).

I actually made this fix a few… _weeks_ ago for private use but I may have uh… completely forgot to actually push and submit the PR (which is now PR #74). Oops. Better late than never though, I guess…?

The source code is freely available for those who want to see exactly what was changed, and/or those who prefer building their own binaries locally.

I've changed the version number to `2.3.1+1.19.2-1.19.1~akemi` (plus the Git commit short SHA-1) in order to clearly differentiate it from the official upstream ExtraSounds versions.

**If you don't want to build ExtraSounds yourself, and you trust me / are comfortable running unofficial binaries, here's a prebuilt jar:** https://developer.akemi.ai/Minecraft/extrasounds-2.3.1+1.19.2-1.19.1~akemi-git-7a58d66.jar

**※ IMPORTANT NOTE:** Minecraft 1.19 is no longer supported with these changes. For more information, see PR #74.
